docs/UserGuide.md

```diff
@@ -3,271 +3,443 @@ layout: page
 
+**Format**: `list [-ATTRIBUTE]`
 
-Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower` Edits the name of the 2nd person to be `Betsy Crower`.
+**Examples**: 
+* Shows a list of all clients and all their information.
+  * `list`
+    
+    ![list](images/list.png)
+    
+*  Shows a list of all clients and their policy number.
+   * `list -policy`
 
-### Locating persons by name: `find`
+[comment]: <> (  insert screenshot)
+*  Shows a list of all clients and their phone number.
+   * `list -phone -policy`
+    
+     ![list phone](images/list-phone-policy.png)
 
-Finds and displays all clients whose field (name, phone, email, address, tags, insurance policy) contains any of the given keywords.
 
-![find_alex_david](images/find_alex_david.png)
+  
+### Search for client contact based on keywords: `find`
 
-![find_alex_&_david](images/find_alex_&_david.png)
+**Purpose**: Finds and displays all clients whose field (name, phone, email, address, tags, insurance policy) contains any of the given keywords.
 
 ![find_alex_&_david_with_filter](images/find_alex_&_david_with_filter.png)
 
-Format: `find FLAG/KEYWORD [& MORE_KEYWORDS] [-ATTRIBUTES]…​`
+**Format**: `find FLAG/KEYWORD [& MORE_KEYWORDS] [-ATTRIBUTES]…​`
 
-* The search is case-insensitive. e.g hans will match Hans.
-* The order of the keywords does not matter.
+* The search is **case-insensitive**. 
+  * E.g. hans will match Hans.
+* The **order of the keywords does not matter**.
 * Only one `FLAG` can be used in each find command.
 * The `FLAG` can only be from: `n`, `p`, `e`, `a`, `t` and `i`, representing name, phone, email, address, tags and insurance policies respectively.
-* The delimiter `&` between keywords is used to search using multiple keywords. e.g. `Aaron & Tan` will return all persons with either `Aaron` or `Tan` in their names.
-* Without the use of delimiter `&`, all keywords following the `FLAG` will be used in the search. e.g. `Aaron Tan` will only return persons with `Aaron Tan` in their names.
-* Clients whose chosen field contains at least one keyword will be returned (i.e. OR search). e.g. Hans & Bo will return Hans Sum, Bo Yang.
-* Filter options are set by typing the attribute name after a dash. e.g. `-phone` filters phone number information of the listed clients
+* The delimiter `&` between keywords is used to search using multiple keywords. 
+  * E.g. `Aaron & Tan` will return all persons with either `Aaron` or `Tan` in their names.
+* Without the use of delimiter `&`, all keywords following the `FLAG` will be used in the search. 
+  * E.g. `Aaron Tan` will only return persons with `Aaron Tan` in their names.
+* Clients with matching keywords in the chosen field will be returned (i.e. OR search). 
```

Comment:
I think it is okay to remove the OR search part because I think the user might not understand what it is.